src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,68 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { useDispatch } from 'react-redux';
+import { actions } from '../../features/currentTodo';
 
-export const TodoModal: React.FC = () => {
-  return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
+type Props = {
+  todo: Todo;
+};
 
-      <Loader />
+export const TodoModal: React.FC<Props> = ({ todo }) => {
+  const [showLoader, setShowLoader] = useState(false);
```

Comment:
```suggestion
  const [showLoader, setShowLoader] = useState(true);
```
to fix modal blinking when user opens the modal

Reply:
+

Reply:
done